2021-02-05 16:43:53,862 INFO    MainThread:9287 [wandb_setup.py:_flush():70] setting env: {}
2021-02-05 16:43:53,863 INFO    MainThread:9287 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-02-05 16:43:53,863 INFO    MainThread:9287 [wandb_setup.py:_flush():70] setting login settings: {'api_key': '6d802b44b97d25931bacec09c5f1095e6c28fe36'}
2021-02-05 16:43:53,863 INFO    MainThread:9287 [wandb_setup.py:_flush():70] setting login settings: {'api_key': '6d802b44b97d25931bacec09c5f1095e6c28fe36'}
2021-02-05 16:43:53,863 INFO    MainThread:9287 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-05 16:43:53,863 INFO    MainThread:9287 [wandb_init.py:_log_setup():318] Logging user logs to /home/wolf/MyCode/HPC/wandb/run-20210205_164353-182k02yi/logs/debug.log
2021-02-05 16:43:53,863 INFO    MainThread:9287 [wandb_init.py:_log_setup():319] Logging internal logs to /home/wolf/MyCode/HPC/wandb/run-20210205_164353-182k02yi/logs/debug-internal.log
2021-02-05 16:43:57,052 INFO    MainThread:9287 [wandb_run.py:_console_start():1399] atexit reg
2021-02-05 16:43:57,053 INFO    MainThread:9287 [wandb_run.py:_redirect():1262] redirect: SettingsConsole.REDIRECT
2021-02-05 16:43:57,054 INFO    MainThread:9287 [wandb_run.py:_redirect():1267] Redirecting console.
2021-02-05 16:43:57,054 INFO    MainThread:9287 [redirect.py:install():213] install start
2021-02-05 16:43:57,055 INFO    MainThread:9287 [redirect.py:install():228] install stop
2021-02-05 16:43:57,055 INFO    MainThread:9287 [redirect.py:install():213] install start
2021-02-05 16:43:57,055 INFO    MainThread:9287 [redirect.py:install():228] install stop
2021-02-05 16:43:57,056 INFO    MainThread:9287 [wandb_run.py:_redirect():1313] Redirects installed.
2021-02-05 16:44:03,732 INFO    MainThread:9287 [wandb_run.py:_atexit_cleanup():1369] got exitcode: 1
2021-02-05 16:44:03,732 INFO    MainThread:9287 [wandb_run.py:_restore():1341] restore
2021-02-05 16:44:03,732 INFO    MainThread:9287 [redirect.py:uninstall():232] uninstall start
2021-02-05 16:44:03,733 INFO    MainThread:9287 [redirect.py:_stop():287] _stop: stdout
2021-02-05 16:44:03,733 INFO    MainThread:9287 [redirect.py:_stop():293] _stop closed: stdout
2021-02-05 16:44:03,733 INFO    stdout    :9287 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-02-05 16:44:03,733 INFO    stdout    :9287 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-02-05 16:44:03,733 INFO    MainThread:9287 [redirect.py:_stop():299] _stop joined: stdout
2021-02-05 16:44:03,733 INFO    MainThread:9287 [redirect.py:_stop():301] _stop rd closed: stdout
2021-02-05 16:44:03,733 INFO    MainThread:9287 [redirect.py:uninstall():236] uninstall done
2021-02-05 16:44:03,733 INFO    MainThread:9287 [redirect.py:uninstall():232] uninstall start
2021-02-05 16:44:03,733 INFO    MainThread:9287 [redirect.py:_stop():287] _stop: stderr
2021-02-05 16:44:03,734 INFO    MainThread:9287 [redirect.py:_stop():293] _stop closed: stderr
2021-02-05 16:44:03,734 INFO    stderr    :9287 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-02-05 16:44:03,735 INFO    stderr    :9287 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-02-05 16:44:03,735 INFO    MainThread:9287 [redirect.py:_stop():299] _stop joined: stderr
2021-02-05 16:44:03,735 INFO    MainThread:9287 [redirect.py:_stop():301] _stop rd closed: stderr
2021-02-05 16:44:03,735 INFO    MainThread:9287 [redirect.py:uninstall():236] uninstall done
2021-02-05 16:44:03,985 INFO    MainThread:9287 [wandb_run.py:_wait_for_finish():1492] got exit ret: file_counts {
  wandb_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 6238
  total_bytes: 6238
}

2021-02-05 16:44:05,990 INFO    MainThread:9287 [wandb_run.py:_wait_for_finish():1492] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 9473
  total_bytes: 9473
}

2021-02-05 16:44:07,600 INFO    MainThread:9287 [wandb_run.py:_show_files():1714] logging synced files
2021-02-05 16:44:07,603 INFO    MainThread:9287 [internal.py:handle_exit():156] Internal process exited
